ONSdigital/dp-frontend-renderer
# Feature/dataset editions page

### What

Added handler and template for dataset editions list page

### How to review

- Pull `feature/dataset-editions-page` branch on dp-frontend-dataset-controller
- Pull the latest `cmd-develop` of the dataset API and make sure the editions end point returns the latest_version url for each edition, e.g. 
```
{
     edition: "2016",
          links: {
               dataset: {
                    id: "95c4669b-3ae9-4ba7-b690-87e890a1c67c",
                    href: "http://localhost:22000/datasets/95c4669b-3ae9-4ba7-b690-87e890a1c67c"
               },
               latest_version: {
                    href: "http://localhost:22000/datasets/95c4669b-3ae9-4ba7-b690-87e890a1c67c/editions/2016/versions/2"
               },
               self: {
                    href: "http://localhost:22000/datasets/95c4669b-3ae9-4ba7-b690-87e890a1c67c/editions/2016"
               },
               versions: {
                    href: "http://localhost:22000/datasets/95c4669b-3ae9-4ba7-b690-87e890a1c67c/editions/2016/versions"
               }
          },
     state: "published"
}
```
- Creating a new instance for the edition should create the latest_version field or it can be added to MongoDB manually (e.g. Robo3T) 

### Who can review
Anyone
